packages/cmf/src/Inject.component.js

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import api from './api';
+
+const getComponentFrom = api.route.getComponentFromRegistry;
+
+function NotFoundComponent({ children, error, ...props }) {
+	console.error(e);
+	return (
+		<div {...props}>
+			<div className="alert alert-danger">{error.message}</div>
+			{children}
```

Comment:
Should we render the chidren element here in case of error ? 
We might not want to render freely that be meant to be render in injected component